### PR TITLE
[FLINK-13037][docs-zh] Translate "Concepts -> Glossary" page into Chi…

### PR DESCRIPTION
**Pull request by highfei2011**
## What is the purpose of the change

*Translate "Concepts -> Glossary" page into Chinese.*

## Brief change log

  - *I modified the glossary.zh.md file .*

## Verifying this change

*I have  verified the changes by starting the build script in preview mode.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no )
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no )
  - The serializers: ( no )
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: ( no )
  - The S3 file system connector: ( no )

## Documentation

  - Does this pull request introduce a new feature? (  no )
  - If yes, how is the feature documented? ( not documented )

@wuchong  @TisonKun  
Please Take A Look .

### CODE REVIEW

File: docs/concepts/glossary.zh.md [hunk 1]
```diff
@@ -25,142 +25,92 @@ under the License.
 
 #### Flink Application Cluster
 
-A Flink Application Cluster is a dedicated [Flink Cluster](#flink-cluster) that only
-executes a single [Flink Job](#flink-job). The lifetime of the
-[Flink Cluster](#flink-cluster) is bound to the lifetime of the Flink Job. Formerly
-Flink Application Clusters were also known as Flink Clusters in *job mode*. Compare to
-[Flink Session Cluster](#flink-session-cluster).
+Flink Application Cluster 是一个专用的 [Flink Cluster](#flink-cluster) ，它仅用于执行单个 [Flink Job](#flink-job) 。 [Flink Cluster](#flink-cluster) 的生命周期与 [Flink Job](#flink-job) 的生命周期绑定在一起。以前，在 *job mode* 中， Flink Application Cluster 也称为 Flink Clusters 。 和 [Flink Session Cluster](#flink-session-cluster) 作对比。
```
Comment (YngwieWang) [inline docs/concepts/glossary.zh.md]:
```suggestion
Flink Application Cluster 是一个专用的 [Flink Cluster](#flink-cluster)，它仅用于执行单个 [Flink Job](#flink-job)。[Flink Cluster](#flink-cluster) 的生命周期与 [Flink Job](#flink-job) 的生命周期绑定在一起。以前，在 *工作模式* 中，Flink Application Cluster 也称为 Flink Clusters。和 [Flink Session Cluster](#flink-session-cluster) 作对比。
```

File: docs/concepts/glossary.zh.md [hunk 2]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Flink Cluster
 
-A distributed system consisting of (typically) one [Flink Master](#flink-master) and one or more
-[Flink TaskManager](#flink-taskmanager) processes.
+一般情况下，Flink 集群是由一个 [Flink Master](#flink-master)和一个或多个 [Flink TaskManager](#flink-taskmanager) 进程组成的分布式系统。
```
Comment (YngwieWang) [inline docs/concepts/glossary.zh.md]:
```suggestion
一般情况下，Flink 集群是由一个 [Flink Master](#flink-master) 和一个或多个 [Flink TaskManager](#flink-taskmanager) 进程组成的分布式系统。
```

File: docs/concepts/glossary.zh.md [hunk 3]
```diff
@@ -25,142 +25,92 @@ under the License.
...
 
 #### Event
 
-An event is a statement about a change of the state of the domain modelled by the
-application. Events can be input and/or output of a stream or batch processing application.
-Events are special types of [records](#Record).
+Event 是对应用程序建模的域的状态更改的声明。它可以同时为流或批处理应用程序的 input 和 output ，也可以单独是 input 或者 output 中的一种 。 Event 是特殊类型的 [Record](#record) 。
```
Comment (YngwieWang) [inline docs/concepts/glossary.zh.md]: Event 是对应用程序建模的域状态更改的声明。它可以是流或批处理应用程序的输入或输出 。Event 是特殊类型的 [Record](#record) 。？

**Comment by highfei2011** [comment]
@YngwieWang Ok,You are right ,I will modify it.

**Comment by YngwieWang** [comment]
> @YngwieWang Ok,You are right ,I will modify it.

@highfei2011 ，昨天我就术语翻译问题在邮件列表征求了意见，社区一致意见是除了[规范](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications)中已翻译的术语以外，专有名词都不翻译，还是保留你原本的翻译方式吧。我昨天的留言给你制造了麻烦，非常抱歉。

**Comment by highfei2011** [comment]
Hi,@YngwieWang ,I have not modify the glossary, just remove some spaces! Thanks for your reminding!

**Comment by highfei2011** [comment]
Here are some [standards](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications).

File: docs/concepts/glossary.zh.md [hunk 4]
```diff
@@ -25,142 +25,92 @@ under the License.
...
-Flink's runtime. Tasks encapsulate exactly one parallel instance of an
-[Operator](#operator) or [Operator Chain](#operator-chain).
+Task 是 [Physical Graph](#physical-graph) 的节点。它是基本的工作单元，由 Flink 的 runtime 来执行。Tasks 正好封装了一个 [Operator](#operator) 或者 [Operator Chain](#operator-chain) 的 *parallel instance*。 
 
 #### Flink TaskManager
 
-TaskManagers are the worker processes of a [Flink Cluster](#flink-cluster). [Tasks](#task) are
-scheduled to TaskManagers for execution. They communicate with each other to exchange data between
-subsequent Tasks.
+TaskManagers 是 [Flink Cluster](#flink-cluster) 的工作进程。[Tasks](#task) 被调度到 TaskManagers 上执行。TaskManagers 相互通信，只为在后续的 Tasks 之间交换数据。
```
Comment (TisonKun) [inline docs/concepts/glossary.zh.md]: It should not be plural when translated.

Comment (TisonKun) [reply docs/concepts/glossary.zh.md]:
```
TaskManager 是 [Flink Cluster](#flink-cluster) 的工作进程。[Tasks](#task) 被调度到 TaskManager 上执行。TaskManager 之间相互通信，以在后续的 Task 之间交换数据。
```

Comment (highfei2011) [reply docs/concepts/glossary.zh.md]: Thanks for your reminding, you are right, I noticed the following instructions: 
image

**Comment by highfei2011** [comment]
Open a new PR .

**Pull request closed without merging**